web-client/public/js/update-app.js

```diff
@@ -477,24 +481,21 @@ const expandLayoutSidebar = () => {
     $(LAYOUT_SIDEBAR_PANEL).addClass("in");
 };
 
-// const toggleLayout = (on, off) => {
-//     if (!$(on).prop("checked")) {
-//         $(on).prop("checked", true);
-//         $(off).prop("checked", false);
-//         $(`${off} span`).removeClass("glyphicon-ok");
-//         $(`${on} span`).addClass("glyphicon-ok");
-//     }
-// };
+const toggleLayout = (on, off) => {
+    if (!$(on).prop("checked")) {
+        $(on).prop("checked", true);
+        $(off).prop("checked", false);
+        $(`${off} span`).removeClass("glyphicon-ok");
+        $(`${on} span`).addClass("glyphicon-ok");
+    }
+};
```

Comment:
Wait—was the intent here to delete the code outright? I seem to recall this was part of @ntran18’s pull request…

Reply:
Nope! So that function toggles between whether two items are checked in the top bar menu. While @ntran18 removed it because the Force Graph and Grid Layout were considered to be buttons that provoked an action rather than part of the graphs state. However, when switching between protein-protein mode and grn mode, this is a state change and the mode should be indicated by the check mark. So I brought back this function and used it for the network type switcher. 

In short it was deleted because it wasn't used for the bug, but I ended up using the pre-existing function in my new code

Reply:
Got it, thanks for clarifying that!